03-lection1/02-accordion/accordion.css

```
@@ -1 +1,51 @@
 /* Стилизация аккордеона */
+
+summary:focus {
```

Вижу у `summary` класс `accordion__summary`, можно было так)
```suggestion
accordion__summary:focus  {
```

---

спасибо, поправил!